### ISSUE TITLE
Pure Data mime info not setup correctly

### ISSUE THREAD

**Issue by jn64** [audinux/fedora-spec#24]
Currently users don't get the association for .pd files.

The wrong file is installed as the mime xml file. Source16 is the pd-README file

https://github.com/audinux/fedora-spec/blob/949633e192de3ed9b339d6078bbee36076bf91b9/pure-data/puredata.spec#L201-L203

It might be some mix up / duplicate with L222-224

https://github.com/audinux/fedora-spec/blob/949633e192de3ed9b339d6078bbee36076bf91b9/pure-data/puredata.spec#L222-L224

I *think* you probably wanted to install `puredata-gui.sharedmimeinfo`  (Source17) as `%{buildroot}%{_datadir}/mime/packages/puredata.xml`, but I'm not 100% sure so I ask first before opening a PR.

**Comment by jn64** [comment]
Tentative patch:

```patch
--- puredata.spec
+++ puredata.spec
@@ -199,8 +199,8 @@
 mkdir -p %{buildroot}%{_datadir}/puredata-gui
 install -m 644 doc/1.manual/1.introduction.txt %{buildroot}%{_datadir}/puredata-gui
 # mime stuff
-mkdir -p %{buildroot}%{_datadir}/mime/packages/
-install -m 644 %{SOURCE16} %{buildroot}%{_datadir}/mime/packages/puredata.xml
+install -m 755 -d %{buildroot}%{_datadir}/mime/packages
+install -m 644 %{SOURCE17} %{buildroot}%{_datadir}/mime/packages/puredata.xml
 
 # hardlink pd-* binaries
 rm -f %{buildroot}%{_bindir}/pd
@@ -219,10 +219,6 @@
 sed -i -e "s/lib/lib64/g" %{buildroot}%{_bindir}/pd-gui
 %endif
 
-# Install mime info
-install -m 755 -d %{buildroot}%{_datadir}/mime/packages
-install -m 644 %{SOURCE17} %{buildroot}%{_datadir}/mime/packages
-
 %files
 %doc README.txt INSTALL.txt
 %license LICENSE.txt
@@ -237,7 +233,7 @@
 %{_libdir}/puredata/doc/7.stuff
 %{_mandir}/man1/pd.1*
 %{_datadir}/pixmaps/puredata.xpm
-%{_datadir}/mime/packages/puredata-gui.sharedmimeinfo
+%{_datadir}/mime/packages/puredata.xml
 
 %files doc
 %{_libdir}/puredata/doc/1.manual/
```

**Comment by ycollet** [comment]
Argh ... OK, thanks.
I will fix this problem ASAP.

**Comment by jn64** [comment]
I realised my patch is not correct, I changed the `%files core` section but now this `puredata.xml` file is duplicated with `%files gui`. I guess the file should only be in 1 sub-package.

I'm still trying to learn about packaging, so I will leave it to you :)

But I can confirm, on my system with currently installed puredata package, if I rename `/usr/share/mime/packages/puredata-gui.sharedmimeinfo`  to `/usr/share/mime/packages/puredata.xml`, and do `sudo update-mime-database /usr/share/mime`, the *.pd files are now recognised in my file manager and I can associate it with Pure Data desktop file.
